lib/opus_support.py

```diff
@@ -1947,7 +1947,25 @@ def parse_form_type(s):
 
     return form_type, form_type_format, form_type_unit
 
-
+def get_single_parse_function(form_type_unit_id):
+    range_func = None

```

Comment:
We don't use the term "range_func" anymore so this one should be called "parse_func' and the next should be "format_func".

Reply:
Likewise just call it "unit_id" not "form_type_unit_id" to match the rest of this file.